src/modules/75-cf/pages/onboarding/OnboardingDetailPage.tsx

```diff
@@ -111,116 +113,152 @@ export const OnboardingDetailPage: React.FC = () => {
+        padding={{ top: 'huge', left: 'xxlarge', right: 'xxxlarge' }}
       >
-        <MultiStepProgressIndicator
-          progressMap={
-            new Map([
-              [0, { StepStatus: firstStepStatus }],
-              [1, { StepStatus: secondStepStatus }],
-              [2, { StepStatus: thirdStepStatus }]
-            ])
-          }
-        />
-      </Layout.Horizontal>
-      <Container className={css.onboardingContainer} height="100%" padding={{ top: 'xxlarge', bottom: 'xlarge' }}>
-        {selectedStep !== STEP.VALIDATE_FLAG ? (
-          <>
-            <Heading level={3} font={{ variation: FontVariation.H3 }} margin={{ bottom: 'large' }}>
-              {getString('cf.onboarding.letsGetStarted')}
-            </Heading>
-            <Heading level={4} font={{ variation: FontVariation.H4 }} margin={{ bottom: 'medium' }}>
-              {getString('cf.onboarding.createFlag')}
-            </Heading>
-
-            {selectedFlag && selectedStep === STEP.SELECT_ENV_SDK && (
-              <Layout.Vertical spacing="medium">
-                <OnboardingSelectedFlag selectedFlag={selectedFlag} />
-                <Divider />
-              </Layout.Vertical>
-            )}
-          </>
-        ) : (
-          <Heading level={3} font={{ variation: FontVariation.H3 }} margin={{ bottom: 'large' }}>
-            {getString('cf.onboarding.validatingYourFlag')}
-          </Heading>
-        )}
-
-        {selectedStep === STEP.CREATE_A_FLAG && (
-          <CreateAFlagView selectedFlag={selectedFlag} setSelectedFlag={setSelectedFlag} />
-        )}
-
-        {selectedStep === STEP.SELECT_ENV_SDK && selectedFlag && (
-          <SetUpYourApplicationView
-            flagInfo={selectedFlag}
-            language={language}
-            setLanguage={setLanguage}
-            apiKey={apiKey}
-            setApiKey={setApiKey}
-            selectedEnvironment={selectedEnvironment}
-            setSelectedEnvironment={setSelectedEnvironment}
-          />
-        )}
-
-        {selectedStep === STEP.VALIDATE_FLAG && selectedFlag && selectedEnvironment && (
-          <ValidateYourFlagView
-            flagInfo={selectedFlag as Feature}
-            language={language as PlatformEntry}
-            apiKey={apiKey as ApiKey}
-            testDone={testDone}
-            setTestDone={setTestDone}
-            environmentIdentifier={selectedEnvironment.identifier}
-          />
-        )}
-      </Container>
-      <Divider />
-      <Layout.Horizontal spacing="small" padding="small">
+        <Layout.Horizontal
+          spacing="small"
+          style={{ justifyContent: 'flex-start', alignItems: 'center' }}
+          padding={{ top: 'xxlarge' }}
+        >
+          <Icon data-testid="cfLogo" name="cf-main" height={30} size={30} />
+          <Text font={{ variation: FontVariation.SMALL_BOLD }} color={Color.GREY_800}>
+            {getString('common.purpose.cf.continuous')}
+          </Text>
+        </Layout.Horizontal>
```

Comment:
I think you should be able to combine the layout, icon and text into a single element if you use the `icon` and `iconProps` props of the `Text` component.
```suggestion
        <Text
          font={{ variation: FontVariation.SMALL_BOLD }}
          color={Color.GREY_800}
          icon="cfLogo"
          iconProps={{ size: 30 }}
        >
          {getString('common.purpose.cf.continuous')}
        </Text>
```

Reply:
The reason for the `Icon` element is that it is custom. If we try to put it in the `Text` element as an icon it doesn't satisfy the IconName:

> Type '"cfLogo"' is not assignable to type 'IconName | undefined'.

Reply:
Haha good spot, resolved